aws_sagemaker_studio/sagemaker_studio_image_build/README.md

```diff
@@ -0,0 +1,20 @@
+# Amazon SageMaker Studio Container Build CLI
+
+This repository contains example notebooks that show how to utilize the [Amazon SageMaker Studio Build CLI](https://pypi.org/project/sagemaker-studio-image-build/). Full usage instructions can be found at the link provided. 
+
+The SageMaker Studio Image Build CLI offers data scientists and developers the ability to build SageMaker compatible docker images directly from their Amazon SageMaker Studio environments without the need to setup and connect to secondary docker build environments. Using the simple CLI, customers now have the ability to easily create container images directly from Amazon SageMaker Studio. 
```

Comment:
```suggestion
The SageMaker Studio Image Build CLI offers you the ability to build SageMaker compatible docker images directly from your Amazon SageMaker Studio environments without the need to setup and connect to secondary docker build environments. Using the simple CLI, you now have the ability to easily create container images directly from Amazon SageMaker Studio. 
```
--> you/your instead of 'customer/data scientist'